markdown/summary.Rmd

```diff
@@ -19,30 +46,29 @@ contact <- data.frame(contact1, contact2, contact3)
+This release by the Information Services Division provides information on the percentage of time spent at home or in a community setting in the last six months of life. This is one of the measures used to support and determine progress towards the Scottish Government’s [“Strategic Framework for Action on Palliative and End of Life Care”](http://www.gov.scot/Resource/0049/00491388.pdf). Data are presented by financial year from `r min(basefile$fy)` to `r max(basefile$fy)`, by NHS Board, Health and Social Care Partnership of residence and for Scotland. 
 
-Note that hospital data and National Records of Scotland deaths data are provisional for 2018/19. This publication will be updated in October 2019 with final figures. Figures for the latest year should therefore be treated with caution as there could be an undercount of hospital admissions, particularly for people who died in the last three months of the year.  This is not expected to have a large impact on these published figures when they are revised in subsequent updates.
+Note that hospital data and National Records of Scotland deaths data are provisional for `r max(basefile$fy)`. This publication will be updated in October `r year(end_date)` with final figures. Figures for the latest year should therefore be treated with caution as there could be an undercount of hospital admissions, particularly for people who died in the last three months of the year.  This is not expected to have a large impact on these published figures when they are revised in subsequent updates.
 
 
 ### Main Points
 
-* In 2018/19, there were 53,168 deaths in Scotland excluding those where an external cause such as unintentional injury was recorded.  For these individuals, 89.2% of their last six months of life was spent either at home or in a community setting, with the remaining 10.8% spent in hospital. This is equivalent to an average of 20 days in hospital in the six months prior to death.
-
-* Over the past nine years, the percentage of time spent at home or in a community setting has gradually increased from 85.3% in 2010/11 to 89.2% in 2018/19.
+* In `r max(basefile$fy)`, there were `r basefile %>% filter(fy == max(fy)) %>% summarise(sum(deaths)) %>% pull(.)` deaths in Scotland excluding those where an external cause such as unintentional injury was recorded.  For these individuals, `r summarise_data(basefile, trend = FALSE) %>% pull(qom)`% of their last six months of life was spent either at home or in a community setting, with the remaining `r 100 - (summarise_data(basefile, trend = FALSE) %>% pull(qom))`% spent in hospital. This is equivalent to an average of 20 days in hospital in the six months prior to death.
```

Comment:
I think the figure in "an average of 20 days in hospital" can also be automated. My code is:

round_half_up(182.5 * ((100 - (summarise_data(basefile, trend = FALSE) %>% pull(qom))) / 100))